fennel.lua

```diff
@@ -752,6 +765,16 @@ local function flatten(chunk, options)
     end
 end
 
+-- module-wide state for metadata
+-- create metadata table with weakly-referenced keys
+local function makeMetadata()
+  return setmetatable({}, METADATA_MT)
```

Comment:
Could we make `METADATA_MT` local to this function? It looks like it isn't used anywhere else.

Reply:
Done!